librz/core/cbin.c

```diff
@@ -4771,3 +5505,85 @@ RZ_API RzCmdStatus rz_core_bin_plugins_print(RzBin *bin, RzCmdStateOutput *state
 	rz_cmd_state_output_array_end(state);
 	return RZ_CMD_STATUS_OK;
 }
+
+static void print_arch(RzBin *bin, RzCmdStateOutput *state, int num, ut64 offset, ut64 size,
```

Comment:
Way too many arguments, what about grouping some in some kind of context? `arch`, `bits`, `machine` are good candidates, for example.